.github/workflows/bindings_nodejs.yml

```diff
@@ -101,10 +101,14 @@ jobs:
           cache-dependency-path: "bindings/nodejs/yarn.lock"
       - name: Corepack
         run: corepack enable
+      - name: Install ziglang
+        uses: goto-bus-stop/setup-zig@v1
```

Comment:
We can't use this action because of the ASF limitation.

Reply:
true.

for now, i think we have several options:
1. change version the base image of ubuntu that support 2.11- glibc.
2. use zig to build such one nodejs binding. here is the references:
    - [how napi use zig to build binding](https://github.com/napi-rs/cross-build/blob/main/.github/workflows/pure-rust-build-on-linux.yml#L47)
    - [relation between zig target version and glibc version](https://github.com/napi-rs/napi-rs/issues/1540#issuecomment-1483103586)
    - [how swc to do so](https://github.com/swc-project/swc/actions/runs/2150531101/workflow)

i am still finding any way to work around. but i am not familiar with github action staff and napi.

it is a little bit hard to trigger the nodejs bindning github action. so i create this PR directly. sorry for thsi

Reply:
> I would prefer not to fix the problem in this manner.

any context here?